R/accessory.R

```diff
@@ -956,10 +956,14 @@ read_contrasts <-
       # Extract blocking factors from 'formula' if available
       blocking <- NA
       formula <- NA
+      make_contrasts_str <- NA
```

Comment:
Here I add the possibility to parse the literal strings from the YAML file, in a similar fashion as we did for the formula.